src/services/Web3.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from "ethers";
+import EventEmitter from "events";
+import Logger from "../helpers/Logger";
+
+/**
+ * Creates a connection to a ws RPC endpoint and exposes events
+ *
+ * @export
+ * @class Web3
+ * @extends {EventEmitter}
+ */
+export default class Web3 extends EventEmitter {
+  private provider: ethers.providers.WebSocketProvider;
+  private logger: Logger;
+
+  constructor(name: string, endpoint: string) {
+    super();
+    this.logger = new Logger(name);
+
+    this.logger.info(`Connecting to ${endpoint}`);
+    this.provider = new ethers.providers.WebSocketProvider(endpoint);
+
+    this.subscribe2NewBlock()
```

Comment:
Idk.. just use the "to" word.. and if strongly believe in this coding slang I also don't care haha :D